packages/ui/v2/core/Stepper.tsx

```diff
@@ -16,7 +21,7 @@ function Stepper<T extends DefaultStep>(props: { href: string; step: number; ste
           <ol role="list" className="ml-8 flex items-center space-x-5">
             {steps.map((mapStep, index) => (
               <li key={mapStep.title}>
-                <Link href={`${href}?step=${index + 1}`} shallow replace>
+                <Link href={props.disableSteps ? "#" : `${href}?step=${index + 1}`} shallow replace>
```

Comment:
Since this is an <a> tag if disabled we set the href to "#" but it will scroll to the top if the user is on a larger screen